js/tests/test_examples.test.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * E2E Tests for plugin demos and examples.
+ */
+
+const { execFile } = require('child_process');
+const path = require('path');
+const { promisify } = require('util');
+const execFileAsync = promisify(execFile);
+
+// Paths
+const PLUGINS = path.resolve(__dirname, '../plugins');
+const EXAMPLES_PATH = path.resolve(__dirname, '../examples');
+
+// Require env vars
+const { OPENAI_API_KEY, COMPOSIO_API_KEY, JULEP_API_KEY, JULEP_API_URL, LISTENNOTES_API_KEY } = process.env;
+console.log(COMPOSIO_API_KEY);
```

Comment:
⚠️ Security Issue: Logging sensitive information. Remove this line to prevent exposing API keys in logs. If logging is needed for debugging, use a safer format like `console.log('COMPOSIO_API_KEY:', COMPOSIO_API_KEY ? '[PRESENT]' : '[MISSING]');`